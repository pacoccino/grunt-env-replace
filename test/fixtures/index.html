<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head>

    <script src="//static-dev.domain.com/service.js"></script>

</head>
<body>

</body>
</html>
