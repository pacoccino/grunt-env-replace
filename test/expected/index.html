<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">
<head>

    <script src="//static-preprod.domain.com/service.js"></script>

</head>
<body>

</body>
</html>
